### PR TITLE
Improve build script

### PR DESCRIPTION
**Pull request by Drapegnik**
## bundles
Include 3dparty libs such as `bulma` and `font-awesome` in separate bundle, and now you no need to include:
```
<link rel="stylesheet" href="../../build/main-page/main-page.css">
<link rel="stylesheet" href="../../node_modules/font-awesome/css/font-awesome.min.css">
<link rel="stylesheet" href="../../node_modules/bulma/css/bulma.css">
```
just add:
```
<link rel="stylesheet" href="../../dist/bundle.css"> <!-- all our styles -->
<link rel="stylesheet" href="../../dist/assets.min.css"> <!-- all 3dparty styles -->
```

## scripts
also now one build script for all, just run `npm run build` for:
* `dist/bundle.css` - all our styles in one file
* `dist/bundle.min.css` - minified production build
* `dist/assets.min.css` - all 3dparty libs in one file
* `fonts/` - FontAwesome fonts

## bulma overrides
you can override bulma default colors and other variables in `src/index.sass`
for more info check [`docs`](http://bulma.io/documentation/overview/customize/)

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -5,36 +5,41 @@ const concatCss = require('gulp-concat-css');
 const stylelint = require('gulp-stylelint');
 const cleanCSS = require('gulp-clean-css');
 
-const buildDir = 'build';
+var config = {
```
Comment (tyndria) [inline gulpfile.js]: why not const, but var?

File: gulpfile.js [hunk 2]
```diff
@@ -5,36 +5,41 @@ const concatCss = require('gulp-concat-css');
...
+};
 
-gulp.task('sass:compile', () => gulp.src('./src/**/*.scss')
-	.pipe(sass().on('error', sass.logError))
-	.pipe(autoprefixer())
-	.pipe(gulp.dest(buildDir))
-);
-
-gulp.task('sass:dev', () => gulp.src('./src/**/*.scss')
+gulp.task('sass:bundle', () => gulp.src('./src/**/*.scss')
```
Comment (tyndria) [inline gulpfile.js]: You know, maybe it's better to create variable for this path './src/**/*.scss'... How do you think?

File: gulpfile.js (hunk 1 above)
Comment (Drapegnik) [inline gulpfile.js]: oops, copy-paste 😮

File: gulpfile.js (hunk 2 above)
Comment (Drapegnik) [inline gulpfile.js]: we can extract `src` in variable in `config`

**Comment by Drapegnik** [comment]
[include 3dparty libs into css bundle](https://trello.com/c/j7OdTP1T/13-include-3dparty-libs-into-css-bundle)

**Pull request merged**